work_queue/src/work_queue_graph_log

```diff
@@ -75,29 +75,12 @@ def read_log_entries(file, fields):
 
             log_entries[record['timestamp']] = record
 
-            record['time_master']   = record['time_status_msgs'] + record['time_internal'] + record['time_application'] + record['time_send'] + record['time_receive']
-            record['bytes_sent'] /= unit_factors['GB']
-            record['bytes_received'] /= unit_factors['GB']
-            record['bytes_transfered'] = record['bytes_sent'] + record['bytes_received']
-
-            try:
-                record['capacity_tasks'] = (record['time_execute'] + record['time_polling']) / record['time_master']
-            except ArithmeticError:
-                record['capacity_tasks'] = 0
-
+            record['time_master']   = record['time_send'] + record['time_receive']
             if(prev_line is None):
-                record['time_master_weighted']  = record['time_master']
-                record['time_execute_weighted'] = record['time_execute_good'] + record['time_polling']
+#                record['time_master_weighted']  = record['time_master']
```

Comment:
Capacity calculation was moved to runtime at the master, so calculating all these weighted and unweighted times is unnecessary. I will continue cleaning up Work Queue Graph Log, it needs some work. I have not looked at it in awhile.